be/src/common/config.h

```diff
@@ -159,6 +159,8 @@ CONF_mInt32(clear_expired_replcation_snapshots_interval_seconds, "3600");
 CONF_String(sys_log_dir, "${STARROCKS_HOME}/log");
 // The user function dir.
 CONF_String(user_function_dir, "${STARROCKS_HOME}/lib/udf");
+// If true, clear udf cache every time be starts
+CONF_Bool(clear_udf_cache_when_start, "true");
```

Comment:
prefer default value to false to avoid download all UDF when start BE

Reply:
Here, all udfs will not be downloaded when be starts, but will only be downloaded on demand during runtime. The default is true and the impact should not be great.

Reply:
This will impact the query performance which has UDF because the JAR will be downloaded from remote.
I suggest we make the default value false to make it compatible with the previous behavior.

Reply:
> This will impact the query performance which has UDF because the JAR will be downloaded from remote. I suggest we make the default value false to make it compatible with the previous behavior.

ok, I have changed the default value to false